grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/Jwt.java

```diff
@@ -0,0 +1,13 @@
+package org.hypertrace.core.grpcutils.context;
+
+import java.util.Optional;
+
+public interface Jwt {
```

Comment:
does this need to be public? for now at least, I think keeping the jwt completely internal to this package would be ideal.

Reply:
Made it package-protected.